### PR TITLE
Adding doc for skipping an exercise

### PR DESCRIPTION
**Pull request by anjalijaiswal**
Added a new page for skipping-exercise in process folder. 
Which will tell user how to skip an exercise.

### CODE REVIEW

File: app/pages/process/skipping-exercises.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+```bash
+exercism skip <LANGUAGE> <SLUG>
+```
+Language here is the track name & slug is the name of the exercise you want to skip. For example, if you want to skip hello-world in Swift language:
+
+```bash
+exercism skip swift hello-world
+```
+
+Now lets say you are ready to work on a skipped exercise, and you want to restore it, issue the command:
```
Comment (kytrinyx) [inline app/pages/process/skipping-exercises.md:19]: If you skipped an exercise you don't need to restore it (restore pulls down your old solutions). The command to fetch the exercise would be:

```
exercism fetch <TRACK_ID> <SLUG>
```

**Comment by kytrinyx** [comment]
Thank you for this.

I'm actually in the process of getting rid of the 'help' site and putting the documentation on the main exercism.io site.

The first step in the process has been to pull in the language-specific documentation into the language pages on the site: http://exercism.io/languages/swift

Maybe we can have a tab on each of the language pages that explains how to skip exercises. Or perhaps it's time to have a "how it works" page on the site.

What do you think?

**Comment by anjalijaiswal** [comment]
I really want to thank you for giving me this chance. It's my first open source contribution. :)

I like the idea of putting help section on main site.  

CLI is common, irrespective of the languages. So i think we should have "how it works" page. Then Language tab will be language specific. After on-boarding the platform a user is going to try many diff languages, so having same commands on each page won't make sense.

**Comment by kytrinyx** [comment]
I agree! Let's have a "how it works" page on the main site, and also a small "help" page that basically says where the support chat is, and suggests opening a PR, and maybe has some troubleshooting suggestions.

I'll pull this in, and then start a "how it works" page. Would you be interested in helping make the how it works page based on the documentation that is currently in "process"? I don't know if all of it should go in there (maybe a quick overview of how it works, and then some tabs that go deeper into the topics).

**Comment by kytrinyx** [comment]
Thanks so much for helping out, and also: welcome to open source! :heart:

**Comment by anjalijaiswal** [comment]
Yeah, i would be very much interested in doing that. :) 
fetching & submitting can be on same page and skipping and other things can have a button like how it is done in help section? Or we can give a sidebar also, but it won't go with current design.

**Comment by kytrinyx** [comment]
I want to avoid the sidebar, except maybe we can use the same kind of tabs like we have in the languages pages.

I'm traveling and won't be home for another couple of days. I'll get the skeleton of this set up and then let you know where I put stuff.

**Comment by anjalijaiswal** [comment]
cool! Thank you again <3 .

**Pull request merged**